packages/primitives/link/src/Root.web.tsx

```diff
@@ -1,11 +1,21 @@
 import React, { FC } from 'react'
 import { normalizeStyle } from 'stili'
-import { TLink } from './types'
+import { TLink, TRel, TTarget } from './types'
 
 const style = normalizeStyle({
   textDecoration: 'none',
 })
 
+const getRel = (list?: TRel[], target?: TTarget) => {
+  if (Array.isArray(list)) {
+    const nonDuplicatesList = new Set(list)
+
+    return [...nonDuplicatesList].reduce((finalValue, currentValue) => `${finalValue} ${currentValue}`, '')
+  }
+
+  return target === '_blank' ? 'noreferrer' : undefined
```

Comment:
Should we set to `noopener` instead of `noreferrer`?

Reply:
just updating from what we discussed:
yes, we're switching for `noopener` because it's a better assumption to the broader usage.